engines/sci/sci.cpp

```diff
@@ -436,7 +436,21 @@ Common::Error SciEngine::run() {
 	}
 
 	if (getGameId() == GID_GK2 && ConfMan.getBool("subtitles") && !_resMan->testResource(ResourceId(kResourceTypeSync, 10))) {
-		suggestDownloadGK2SubTitlesPatch();
+		suggestDownloadPatch(_("GK2 has a fan made subtitles, available thanks to the good persons at SierraHelp.\n\n"
```

Comment:
> GK2 has a fan made subtitles, available thanks to the good persons at SierraHelp

This sounds strange to my ears as a native-speaker of American English. Two things:

* "a […] subtitles": the first word implies singular, the second is plural. I think it should just be "GK2 has fan made subtitles"
* The good persons at SierraHelp" just sounds awkward. Maybe just "hosted at SierraHelp"?